docs/_content/standaard/index.md

```diff
@@ -65,7 +68,7 @@ De API’s zijn geïnspireerd op RGBZ en imZTC waarin objecten, gegevens en de o
 Het gegevensmodel voor de ZGW API’s benadert het totaalbeeld wat in RGBZ en imZTC wordt geschetst nog het meest. Daar waar in RGBZ resources uit verschillende bronnen en modellen samenkomen is dat in de API’s voor Zaakgericht Werken opgesplitst. Er is een Catalogus API voor de ZTC en Zaken API voor zaken een documenten API voor Document (Informatieobjecten) en een Besluiten API voor Besluiten.
 
 Het diagram toont geen ObjectTypen uit de Autorisatie API en de Notificaties API omdat deze geen (modelleerbare) relaties hebben met ObjectTypen uit de ander API's.
-[![Overkoepelend gegevensmodel](./ZGW API's.png){:width="1200px"}](./ZGW API's.png "Gegevensmodel, klik voor grote versie")
+[![Overkoepelend gegevensmodel](./ZGW API's 1.1.0.png){:width="1200px"}](./ZGW API's 1.1.0.png "Gegevensmodel, klik voor grote versie")
```

Comment:
Hmm, dit is lastig ding. De ZGW API's hebben overkoepelend geen versie nummer.

Reply:
Om toch een verschil te kunnen maken in versies van het model, kunnen we er een datum oid aan hangen?

Reply:
@joeribekker @michielverhoef 
OK, die versie moet er uit. Dat is verwarrend. 

Zonder aanvulling (b.v. datum) in de naamgeving vervangen we dan steeds het overkoepelende model. 

Het overkoepelende model wijzigt zodra er iets verandert in de op het niveau van resources van de onderliggende API's of relaties tussen de resources van alle API's. Dit houdt gelijke tred met het opleveren van nieuwe of gewijzigde  functionaliteit, zoals de RC van de Klantinteractie API's.

Willen we eerdere 'versies' van dat overkoepelende model aanbieden?

Wat daarbij verder meespeelt:
- Als het meer dan één 'versie aanbieden, wanneer doen we dat dan (alleen bij major en of minor releases van API's, of ook op andere momenten)?;
- Moet er een (andere) plek zijn waar mensen de gevolgen voor het gegevensmodel van actuele ontwikkelingen kunnen bekijken, en op welke manier bieden wij dat aan? Wie heeft daar wat aan?
- Hoe vaak gaan we het complete gegevensmodel verversen?